packages/mobile/src/screens/settings-screen/AccountVerificationScreen.tsx

```diff
@@ -250,6 +263,25 @@ export const AccountVerificationScreen = () => {
     )
   }
 
+  const handleTikTokPress = () => {
+    if (!handle) return
+    onVerifyButtonPress()
+    dispatch(oauthActions.setTikTokError(null))
+    dispatch(oauthActions.tikTokAuth())
+    track(
+      make({
+        eventName: EventNames.SETTINGS_START_TIKTOK_OAUTH,
+        handle
+      })
+    )
+  }
+
+  useEffect(() => {
+    if (abandoned) {
+      setStatus('')
```

Comment:
Nit, but is there a better value for this?

Reply:
Yessir, can use Status.Idle